src/common/perf_counters.cc

```diff
@@ -190,7 +190,14 @@ void PerfCounters::dec(int idx, uint64_t amt)
   ceph_assert(!(data.type & PERFCOUNTER_LONGRUNAVG));
   if (!(data.type & PERFCOUNTER_U64))
     return;
-  data.u64 -= amt;
+  if (amt > data.u64) { // would wrap
+    assert(false); // Assert in debug builds
+    lderr(m_cct) << "WARNING: PerfCounters::dec() " << data.u64 << " - "
```

Comment:
might want to use `__func__` ?

Reply:
__func__ outputs just 'dec' which I thought was less informative than 'PerfCounters::dec()' but sure :)